styles.css

```diff
@@ -0,0 +1,745 @@
+body {
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    font-family: "Bakbak One", Arial, sans-serif;
+    background: #0D0D15;
+}
+
+/* Bakbak One for headings */
+.bakbak-one-regular {
+    font-weight: 400;
+    font-style: normal;
+}
+
+/* Poppins for body text */
+.poppins-medium {
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-style: normal;
+}
+
+/* Header */
+.header {
+    padding: 32px;
+    /* Removed width: 100% as block elements automatically take full width */
+    /* background: #0D0D15; WILL INHERIT IT FROM BODY*/
+}
+
+.logo {
+    width: 188px;
+    height: 42px;
+}
+
+.logo img {
+    padding: 35px;
+    top: 1px;
+}
+
+.main-nav {
+    vertical-align: top;
+    width: auto;
+}
+
+.main-nav__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.main-nav__link {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 1.5;
+    color: #F5F8FB;
+    text-decoration: none;
+}
+
+.main-nav__list-item {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
```

Comment:
These properties are redundant since they are already defined for the` .main-nav__link`, which is inside this item.

Reply:
Not fixed :(